src/templates/finops-hub/modules/dataFactory.bicep

```diff
@@ -5044,6 +5183,13 @@ resource pipeline_ExecuteIngestionETL 'Microsoft.DataFactory/factories/pipelines
       timestamp: {
         type: 'string'
       }
+      clusterstatusvalue: {
```

Comment:
This should be camel case everywhere `clusterStatusValue`. Maybe even name it inline with the other variable: `dataExplorerStatus`.